CONTRIBUTING.rst

```diff
@@ -46,8 +46,34 @@ Every issue will get a **unique issue number**.
 
 **Step 2**: Create a local branch that corresponds to the issue. There are utility scripts to help you with this::
 
-  $ ./tools/create-patch-branch <unique issue number> '<component> #<issue number>: <short_change_message>'
-  $ ./tools/create-feature-branch <unique issue number> '<component> #<issue number>: <short_change_message>'
+  $ ./tools/create-patch-branch <unique issue number> '<short_change_message> #<issue number>: '
```

Comment:
i'd like to have the component in there, separated by a colon. what do others think?

"clients: my change #123"

Reply:
The component is redundant with the label and generate some repetition in the final release notes.

Reply:
For me the important part is the commit message, there the component should be mentioned and the guide already specifies it like this. In the branch name just `$ ./tools/create-patch-branch <unique issue number> '<short_change_message>` is sufficient. There is no point to have the #issue number at the end, it is already mentioned before and the # is filtered anyway as it is a branch name.